ui/components/component-library/text/text.tsx

```diff
@@ -43,6 +43,7 @@ export const Text: TextComponent = React.forwardRef(
       ellipsis,
       className = '',
       children,
+      isBrandEvolution, // Phase 1: Brand Evolution Typography
```

Comment:
In our last internal OH @brianacnguyen called out that we may want to reinforce that this is for brand evolution and [shouldn't be used otherwise](https://github.com/consensys-vertical-apps/metamask-portfolio/pull/1029/files#diff-ddc1067f080930bc48865f4660316a14b21bbc9f9ba85975ce8479342b983d2eR352)

```suggestion
      isBrandEvolution, // Enables Brand Evolution Typography do not use unless you are working on the brand evolution
```